README.md

```diff
@@ -17,6 +17,9 @@ Please see [`Prerequisites for Driver Installation`](https://github.com/IBM/ibm-
 ## Installation
 
 ### Install the operator
+
+> **Note**: You can install the operator via OperatorHub in OpenShift web console directly.
```

Comment:
@rivkap  please approve the text.
Maybe something like to install in openshift go to openshift GUI and install it from the operatorhub. 
?

Reply:
Note: The operator can be installed directly from the RedHat OpenShift web console, through the OperatorHub.